revolv/project/tests/tests.py

```diff
@@ -242,3 +244,92 @@ class ScrapeTest(TestCase):
     def test_scrape(self):
         result = scrape.delay()
         self.assertTrue(result.successful())
+
+
+class DonationAjaxTestCase(CreateTestProjectMixin, TestUserMixin, TestCase):
+    VALIDATE = 'payment/validate'
+    SUBMIT = 'payment/submit'
+
+    def setUp(self):
+        TestUserMixin.setUp(self)
```

Comment:
I didn't use `super` because of multiple inheritance. I feel like this makes it more explicit anyway. I can change it if you'd like.